python/python/glide/async_commands/server_modules/json.py

```diff
@@ -1,18 +1,18 @@
 # Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
-"""module for `RedisJSON` commands.
+"""module for `JSON` commands.
 
     Examples:
 
-        >>> from glide import json as redisJson
-        >>> import json
+        >>> from glide import json
+        >>> import json as jsonpy
```

Comment:
I wouldn't recommend aliasing standard libraries.  Users won't want to refactor their code for our client.

Reply:
I will cgange it once I change the module name